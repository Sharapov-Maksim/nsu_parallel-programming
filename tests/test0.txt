8
-2 22 33 88 0 4 -32 2