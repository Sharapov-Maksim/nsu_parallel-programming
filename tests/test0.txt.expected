8
-32 -2 0 2 4 22 33 88 
